Make docs is failing in GA

The make docs tests in GA are constantly failing. 
Example: https://github.com/OwnCA/ownca/runs/7416704390?check_suite_focus=true


```shell
Run make docs
[7](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:8)
sphinx-apidoc -o  docs/source/ ownca/
[8](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:9)
Traceback (most recent call last):
[9](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:10)
  File "/opt/hostedtoolcache/Python/3.9.13/x64/bin/sphinx-apidoc", line 5, in <module>
[10](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:11)
    from sphinx.ext.apidoc import main
[11](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:12)
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/sphinx/ext/apidoc.py", line 29, in <module>
[12](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:13)
    from sphinx.cmd.quickstart import EXTENSIONS
[13](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:14)
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/sphinx/cmd/quickstart.py", line 44, in <module>
[14](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:15)
    from sphinx.util.template import SphinxRenderer
[15](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:16)
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/sphinx/util/template.py", line 18, in <module>
[16](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:17)
    from sphinx.jinja2glue import SphinxFileSystemLoader
[17](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:18)
  File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/sphinx/jinja2glue.py", line 15, in <module>
[18](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:19)
    from jinja2 import FileSystemLoader, BaseLoader, TemplateNotFound, \
[19](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:20)
ImportError: cannot import name 'contextfunction' from 'jinja2' (/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/jinja2/__init__.py)
[20](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:21)
make: *** [Makefile:43: docs] Error 1
[21](https://github.com/OwnCA/ownca/runs/7416704482?check_suite_focus=true#step:8:22)
Error: Process completed with exit code 2.

```

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -1,4 +1,7 @@
 -i https://pypi.org/simple
+cryptography==3.4.8
+voluptuous==0.12.2
+
 alabaster==0.7.12
 babel==2.10.3; python_version >= '3.6'
 certifi==2022.6.15; python_version >= '3.6'

